omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -290,6 +290,18 @@
     end
   end
 
+  block "Remove .pyi files" do
+    # We don't need them at runtime
+
+    pattern = "#{install_dir}/embedded/lib/python#{python_version}/site-packages/**/*.pyi"
+
+    if windows_target?
+      pattern = "#{python_3_embedded}/Lib/site-packages/**/*.pyi"
+    end
```

Comment:
Could be a follow-up, but given all the manipulations that happen inside `site-packages` it might be worth computing the path to it in a single place to avoid requiring all these similar conditionals.

Reply:
Agreed! https://github.com/DataDog/datadog-agent/pull/34496/commits/bf2a757c96e17d1ea1bc743d36af41d66caaa8f1 What do you think?

Reply:
hahaha I agree, I pushed that fix already :)